apis/beacon/blobs/blobs.yaml

```diff
@@ -0,0 +1,62 @@
+get:
+  operationId: getBlobs
```

Comment:
```suggestion
  operationId: getBlobsSidecar
```

Reply:
Yeah I had that originally but dropped it. There was a suggestion a while ago to remove it, and I think it's a fair enough reason, although I also think `blob_sidecars` make sense too given it's the structure we return and consistent with the spec name. I don't have a strong opinion here though.

https://github.com/ethereum/beacon-APIs/pull/286#issuecomment-1379756410

Reply:
i think in previous version the comment made sense as blobs were sort of different from the sidecars aggregate object, but not blobSideCar is an independent object, i means its just not taxing on the mind to understand in a glance what they return.
but yes i wont strongly push it either but weakly in favor of change now

Reply:
Thanks @g11tech, I think that sounds reasonable to me, so: 
- operationId: getBlobSidecars
- path: `/eth/v1/beacon/blob_sidecars/{block_id}`

Any other thoughts on this @mcdee @rolfyone?

Reply:
I'm impartial on this one :)

Reply:
Thanks for the inputs, I'll wait another few days before updating this to see if anyone else have an opinion on this.

Reply:
Updated to `blob_sidecars` in a6d1c360991676c36cc5691a17d8ac1c17813c33